### ISSUE TITLE
Bluetooth and universal clipboard

### ISSUE THREAD

**Issue by warc0s** [hackintosh-club/intel-nuc10#128]
Is there anyone else having issues with Bluetooth and Universal Clipboard?
Regarding Bluetooth, it connects perfectly to wireless headphones, but I can't connect it to either a Bluetooth keyboard or mouse, both from Logitech, in case that might be relevant.
Additionally, the Universal Clipboard isn't working either, and as far as I can see, it should be functioning.

I've tried both Sonoma and Ventura, and I have exactly the same issue on both OS. Any ideas on how to solve this? Thanks.

**Comment by apollohackintosh** [comment]
> Is there anyone else having issues with Bluetooth and Universal Clipboard? Regarding Bluetooth, it connects perfectly to wireless headphones, but I can't connect it to either a Bluetooth keyboard or mouse, both from Logitech, in case that might be relevant. Additionally, the Universal Clipboard isn't working either, and as far as I can see, it should be functioning.
> 
> I've tried both Sonoma and Ventura, and I have exactly the same issue on both OS. Any ideas on how to solve this? Thanks.

Did you change the order of the kexts in the EFI?

**Comment by warc0s** [comment]
No, I haven't altered the order of the kexts in the EFI; I've left them as they were uploaded. Should I arrange them in a specific way? I don't recall reading anything about needing to do that.

**Comment by apollohackintosh** [comment]
> No, I haven't altered the order of the kexts in the EFI; I've left them as they were uploaded. Should I arrange them in a specific way? I don't recall reading anything about needing to do that.
No you don't need to do that but best to double check it's in the same order.
